### DISCUSSION TITLE
Notification noise and usability of large discussions (e.g. feature feedback)

### DISCUSSION THREAD

**Discussion by mxmeinhold** [community/community#12952] [Discussions]
If discussions are going to be the usual platform for feature announcements and feedback, I've got a short wishlist of features.

* subscribe to specific answers
Instead of subscribing to the entire discussion, which has gotten me over 100 issues in the last 2 days, I'd like to be able to subscribe only to my 'answer', or to specific ones that I'm interested in.

* answer filtering and search
Clicking `load more` and ctrl-f'ing down the page is workable but not really the best ux. I'd like to at least be able to filter to `answers I am participating in`, and search might be handy too.

**Comment by suprstarrd** [comment]
These would be awesome. Add to this: can we get "newest by default"? That would allow new suggestions and feedback to easily flow through, instead of pushing older comments to the top.

**Comment by richard-rance** [comment]
Searching for answers is important for large feedback discussions like [this one](https://github.com/github/feedback/discussions/12341). There are a lot of duplicate answers because you can't check to see if someone else said something similar. Also, I know I left an "answer" but don't have a way to find it without clicking load more many times.

**Comment by borekb** [comment]
Subscribing only to specific threads would be great! I just had to unsubscribe from https://github.com/orgs/github-community/discussions/16925 as I was just overwhelmed with notifications by "random" people trying the new GitHub feature. It means that I will miss answers to specific threads I'm interested in.

**Comment by CanRau** [comment]
Yes please I was just about to open a new discussion for this as I had some occasions where I only wanted to subscribe to one answer and its replies in a discussion and not necessarily the whole discussion 😇

And as already mentioned in [#3951](https://github.com/orgs/community/discussions/3951), subscribing to categories would be nice
